io_directory_example/settings/lens_system3_config.yml

```diff
@@ -44,7 +44,7 @@ mask:
   dec_at_xy_0:
   - -2.4
   - -2.4
-  custom_mask:
+  custom_mask:  #Alternating Pixel Mask (1 represent masking )
```

Comment:
Do you mean 1 means masked pixel? The usual convention is for zero to mean masked region. Are you inverting the mask before sending to lenstronomy? If so, please invert this convention.

Reply:
Yes I did  invert the mask before sending to lenstronomy, I initially choose this convention as this was the output given by deepCR. I have inverted the convention